### PR TITLE
CANS-1184  Prepare data for regression test

### PR DESCRIPTION
**Pull request by vladimirPe**
## JIRA
https://osi-cwds.atlassian.net/browse/CANS-1184
## Description

Preparation of initial data for regression test

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [x] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.

### CODE REVIEW

**Comment by vladimirPe** [comment]
Jenkins retest this please

File: spec/regression/case_worker_scenarios_spec.rb [hunk 1]
```diff
@@ -200,6 +201,66 @@
...
+    number_of_6_to_21_needed = MIN_INITIAL_ASSESSMENTS - numbers[AGE_6_21]
+    # Create assessments for age 0-5
+    create_assessments number_of_0_to_5_needed, true if number_of_0_to_5_needed > 0
+    # Create assessments age 6-21
+    create_assessments number_of_6_to_21_needed, false if number_of_6_to_21_needed > 0
+  end
+
+  def numbers_of_completed
+    visit '/'
+    @staff_dash.client_link(CLIENT_NAME).text eq(CLIENT_NAME)
```
Comment (DmitriyRudenko) [inline spec/regression/case_worker_scenarios_spec.rb]: did you forget `expect(..` here?

Comment (vladimirPe) [reply spec/regression/case_worker_scenarios_spec.rb]: No I did not

File: spec/page_objects/client_profile.rb [hunk 2]
```diff
@@ -23,4 +28,32 @@ def go_to_recently_updated_assessment(assessment_date)
     end
     assessment.click
   end
+
```
Comment (DmitriyRudenko) [inline spec/page_objects/client_profile.rb]: do we really need those extra "click.." methods?

Comment (vladimirPe) [reply spec/page_objects/client_profile.rb]: I'm not sure, someone put it there, better to ask the author

File: spec/regression/case_worker_scenarios_spec.rb (hunk 1 above)
Comment (vladimirPe) [reply spec/regression/case_worker_scenarios_spec.rb]: have deleted the line

File: spec/page_objects/client_profile.rb (hunk 2 above)
Comment (DmitriyRudenko) [reply spec/page_objects/client_profile.rb]: I mean methods:

`click_switch_to_comparison`, `click_age_0_5_button`, `click_age_6_21_button` which are part of your changes

not important actually but it'll reduce amount of code

**Pull request merged**